com.ibm.wala.core/src/com/ibm/wala/ipa/slicer/SDG.java

```diff
@@ -650,6 +677,7 @@ public boolean hasEdge(Statement src, Statement dst) {
       case HEAP_RET_CALLER:
       case METHOD_ENTRY:
       case METHOD_EXIT:
+      case CATCH:
```

Comment:
I believe so; I am surprised it was not caught by WALA's own tests, but I guess this particular case does not come up in typical uses of hasEdge()...